src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -540,21 +540,16 @@ public static void exitGameIfFinished() {
    * todo, replace with something better
    * Get the chat for the game, or null if there is no chat.
```

Comment:
Javadoc still states `null` may be returned.

Reply:
Fixed in 47cb6ff